.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,9 @@
-<!--
-Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+## What it solves
 
-This PR template was copied from https://github.com/testing-library/react-testing-library/blob/main/.github/PULL_REQUEST_TEMPLATE.md
+## How this PR fixes it <!-- A technical description of what have changed -->
```

Comment:
I thought we wanted to ditch the comments? 😄 
I think the title alone is clear enough.

Reply:
hehe, done 😁